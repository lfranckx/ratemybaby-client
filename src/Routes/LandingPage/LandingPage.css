#landing-header {
    background: -webkit-linear-gradient(rgb(169, 9, 9) , rgb(74, 38, 27));
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent; 
    font-size: 4rem;
    opacity: 0.9;    
    margin: 100px 0 150px;
}

.register {
    background-color: rgb(249, 103, 103);
    background-image: linear-gradient(to right, rgb(247, 87, 87) , rgb(249, 147, 103));    
    color: white;
    padding: 15px 90px;
    border-radius: 40px;
    font-size: x-large;
    font-weight: bold;
    width: 100%;
    box-shadow: 1.5px 2px 3px grey;
}

.register:active {
    background-color: rgb(249, 103, 103);
    background-image: linear-gradient(to right, rgb(247, 87, 87) , rgb(247, 87, 87));    
}

/* MEDIA QUERIES */
@media (max-width: 700px) {
    #landing-header {
        font-size: 2rem;
        margin-bottom: 80px;
    }
    
    .register{
        padding: 7.5px 45px;
        border-radius: 40px;
        font-size: 15px;
    }
}

@media (max-width: 400px) {
    
}