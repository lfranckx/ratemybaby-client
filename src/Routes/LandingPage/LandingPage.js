import React, { Component } from 'react'
import './LandingPage.css'
import { Link } from 'react-router-dom'

class LandingPage extends Component {

    render() {
        return (
            <section className="landing-section">
                <h2 id="landing-header">Upload your baby, the world rates it.</h2>
                <div>
                    <Link id='registerlink' className="register" to='/register'>
                        Sign Up
                    </Link>
                </div>
            </section>
        )
    }
}

export default LandingPage