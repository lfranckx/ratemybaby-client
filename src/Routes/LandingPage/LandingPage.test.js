import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from './LandingPage';

describe(`LandingPage component`, () => {

    it(`renders page without crashing`, () => {
        const wrapper = shallow(<LandingPage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it(`links to sign up page`, () => {
        const wrapper = shallow(<LandingPage />);
        expect(wrapper.find('#registerlink').props().to).toBe('/register');
    });
})