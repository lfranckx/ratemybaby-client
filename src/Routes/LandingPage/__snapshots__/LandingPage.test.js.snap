// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders page without crashing 1`] = `
<section
  className="landing-section"
>
  <h2
    id="landing-header"
  >
    Upload your baby, the world rates it.
  </h2>
  <div>
    <Link
      className="register"
      id="registerlink"
      to="/register"
    >
      Sign Up
    </Link>
  </div>
</section>
`;
